wurst/systems/units/hostiles/Hostile.wurst

```diff
@@ -36,9 +37,6 @@ public abstract class Hostile extends UnitEntity
         for i = 1 to numCorpses
             createCorpse(players[PLAYER_NEUTRAL_AGGRESSIVE], UNIT_MEAT, pos, randomAngle())
 
-        let iter = drops.iterator()
-        for drop from iter
-            createItem(drop, pos)
-        iter.close()
+        drops.each(drop -> createItem(drop, pos))
```

Comment:
Do you need to destroy `drops` after this?

Reply:
`.each` will cleanup the list unless it is owned (which this list is not)